<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>1640</width>
                <height>1167</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>MainWindow</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <widget class="QCustomPlot" name="price_graph" native="true">
                <property name="geometry">
                    <rect>
                        <x>40</x>
                        <y>0</y>
                        <width>1080</width>
                        <height>800</height>
                    </rect>
                </property>
            </widget>
            <widget class="QPushButton" name="bx_buy">
                <property name="geometry">
                    <rect>
                        <x>1150</x>
                        <y>800</y>
                        <width>150</width>
                        <height>60</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Add Buy Pressure</string>
                </property>
            </widget>
            <widget class="QPushButton" name="bx_sell">
                <property name="geometry">
                    <rect>
                        <x>1370</x>
                        <y>800</y>
                        <width>150</width>
                        <height>60</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Add Sell Pressure</string>
                </property>
            </widget>
            <widget class="QCustomPlot" name="heatmap" native="true">
                <property name="geometry">
                    <rect>
                        <x>1139</x>
                        <y>-1</y>
                        <width>471</width>
                        <height>801</height>
                    </rect>
                </property>
            </widget>
            <widget class="QLabel" name="lb_total_orders">
                <property name="geometry">
                    <rect>
                        <x>1150</x>
                        <y>875</y>
                        <width>150</width>
                        <height>60</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Total Orders:</string>
                </property>
            </widget>
            <widget class="QLabel" name="lb_best_bid">
                <property name="geometry">
                    <rect>
                        <x>1150</x>
                        <y>935</y>
                        <width>150</width>
                        <height>50</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Best Bid:</string>
                </property>
            </widget>
            <widget class="QLabel" name="lb_spread">
                <property name="geometry">
                    <rect>
                        <x>1370</x>
                        <y>875</y>
                        <width>141</width>
                        <height>60</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Spread:</string>
                </property>
            </widget>
            <widget class="QLabel" name="lb_best_ask">
                <property name="geometry">
                    <rect>
                        <x>1370</x>
                        <y>935</y>
                        <width>150</width>
                        <height>60</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Best Ask:</string>
                </property>
            </widget>
            <widget class="QLabel" name="lb_bid_depth">
                <property name="geometry">
                    <rect>
                        <x>1150</x>
                        <y>995</y>
                        <width>150</width>
                        <height>50</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Best Bid Depth:</string>
                </property>
            </widget>
            <widget class="QLabel" name="lb_ask_depth">
                <property name="geometry">
                    <rect>
                        <x>1370</x>
                        <y>995</y>
                        <width>150</width>
                        <height>60</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Best Ask Depth:</string>
                </property>
            </widget>
            <widget class="QCustomPlot" name="volume" native="true">
                <property name="geometry">
                    <rect>
                        <x>40</x>
                        <y>850</y>
                        <width>1080</width>
                        <height>200</height>
                    </rect>
                </property>
            </widget>
        </widget>
        <widget class="QMenuBar" name="menubar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>1640</width>
                    <height>43</height>
                </rect>
            </property>
        </widget>
        <widget class="QStatusBar" name="statusbar"/>
    </widget>
    <customwidgets>
        <customwidget>
            <class>QCustomPlot</class>
            <extends>QWidget</extends>
            <header>qcustomplot.h</header>
            <container>1</container>
        </customwidget>
    </customwidgets>
    <resources/>
    <connections/>
</ui>
